coral/.husky/pre-commit

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+STAGED_FILES=$(git diff --staged --name-only)
+
+if echo "$STAGED_FILES" | grep -q coral;
+then
+  echo 'Files on coral staged'
+    cd coral || return
+    pnpm lint-staged
+    pnpm test --bail
+fi
```

Comment:
Missing newline.

Reply:
Will remove! But shouldn't prettier have catched that one?

Reply:
The script fails with a newline `Command failed with ENOENT`